py/make_root_pointers.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
Please rename this file to `makerootpointers.py` for consistency with other scripts.

Reply:
Normally, I am in favor of consistency, but I have a really hard time reading words that are squished together like this. My brain moves/adds/removes letters until it finds a word that it recognizes, but it is rarely what is actually written. So, I was hoping to start a new trend of being more readable.

Reply:
OK, let's keep it as `make_root_pointers.py`.